milli/src/error.rs

```diff
@@ -216,6 +218,7 @@ impl fmt::Display for UserError {
                 )
             }
             Self::InvalidFilter(error) => error.fmt(f),
+            Self::InvalidAscDescSyntax { name } => write!(f, "invalid asc/desc syntax {}", name),
```

Comment:
Not sure about this but the error message is not the one we want.
https://github.com/meilisearch/specifications/pull/68/files#diff-a3090c4f9494d7b6ef78610abf435f2ce4fcd50b118ec82f81701fbef0be773aR203

Reply:
You're right!